author/ietf/topics/references.adoc

```diff
@@ -2,186 +2,105 @@
-<3> References are recognized in `relref` as well as `xref` elements.
-<4> Metanorma-IETF differentiates between bibliographic references and crossreferences to other anchors within the document.
-<5> The bibliographic headers need to be provided as above, and its titles are expected to be "Normative References" and "Informative References"; Metanorma will look for those titles specifically in order to insert the references it identifies from the file. However, no XML content is expected to be provided under each heading, and any XML content that is provided will be ignored.
-
-[[external-ref-lookup]]
-== Lookup of external references
-
-In order to speed the lookup of references on the http://xml.resource.org/public/rfc website for external
-references, a cache of references is built the first time the gem is run, in the user's home directory:
-`~/.metanorma-ietf-biblio-cache.json`. This maps all the canonical anchors for external references as defined
-by the IETF, such as `RFC2119` or `CCITT.E163.1988`, to the URLs that their RFC reference is stored on.
-References are detected for replacement in the Asciidoc document by matching the `references@anchor`
-attribute to one of those canonical anchors; the content of the local RFC XML reference is not checked.
-
-(For rebuilding the cache, see <<caches>>.)
-
-Rather than hand crafting RFC XML references for RFC documents, or other references hosted at `http://xml.resource.org/public/rfc/`,
-you need only create a dummy `<reference>` element containing the IETF-defined anchor for that citation. In postprocessing, any
-references that are hosted at xml.resource.org will be identified by their anchor, and replaced by entities or XML includes,
-for RFC XML v2 and v3 respectively. (The complete list of authoritative
-RFC XML bibliographies is available from https://xml2rfc.tools.ietf.org , online and in zipped form.)
-
-Because the content of any externally defined `<reference>` element is overwritten with an entity or include,
-you do not need to provide a full reference. You can supply a minimal reference like `<reference anchor="RFC2119"/>`,
-but note that such a reference is invalid in the RFC XML schema, and the gem will report a missing element during processing.
-(The document will still be processed successfully.) To prevent any validation error reports, the minimal syntactically valid reference is:
-
-[source,xml]
---
-<reference anchor="RFC2119">
-  <front>
-    <title/>
-    <author/>
-    <date/>
-  </front>
-</reference>
---
-
-If you wish to cite a specific version of an Internet-Draft, you will need to include the `seriesInfo` element
-in the reference that identifies the specific version; the anchor is the same for all internet drafts. For example:
+* [[[ref1,1]]] MacDonald, Ian. 1991. _The New Shostakovich._ Boston: Northeastern University Press.
+
+[%bibitem]
```

Comment:
There is something clearly wrong with this example, the type is `googlebooks` but the link is to Amazon. The `:contributor:` attribute is empty... etc.